packages/kit/src/runtime/client/client.js

```diff
@@ -1498,6 +1500,15 @@ async function load_data(url, invalid) {
 	return server_data;
 }
 
+/**
+ * @param {unknown} error
+ * @param {import('types').ClientRequestEvent} event
```

Comment:
I wonder if the name of this should reflect `LoadEvent` somehow

Reply:
Yeah I felt the same, but didn't come up with a good name